testing/describes.js

```diff
@@ -875,3 +875,14 @@ function createAmpElement(win, opt_name, opt_implementationClass) {
   element.classList.add('i-amphtml-element');
   return element;
 }
+
+/**
+ * Run a test suite multiple times based on variable inputs.
+ * @param {!Array} values
+ * @param {func} func
+ */
+export function testCases(values, func) {
```

Comment:
Huh?

Reply:
I use this in test-amp-mustache like so that i can reuse all the tests and just run them using template creation with <template> in one test run and <script> in the other.

e.g.

`testCases(['script', 'template'], templateType => {
   beforeEach(
      document.createElement(templateType);
   )
})`

Reply:
Did you try using `describes.repeated`?